src/controllers/itemDetails/index.js

```diff
@@ -470,43 +497,18 @@ define(['loading', 'appRouter', 'layoutManager', 'connectionManager', 'userSetti
         var imgUrl;
         var hasbackdrop = false;
         var itemBackdropElement = page.querySelector('#itemBackdrop');
-        var usePrimaryImage = item.MediaType === 'Video' && item.Type !== 'Movie' && item.Type !== 'Trailer' ||
-            item.MediaType && item.MediaType !== 'Video' ||
-            item.Type === 'MusicAlbum' ||
-            item.Type === 'Person';
 
-        if (!layoutManager.mobile && !userSettings.detailsBanner()) {
+        if (!layoutManager.mobile && ((!userSettings.enableBackdrops() && !userSettings.detailsBanner()) || (userSettings.enableBackdrops() && !userSettings.detailsBanner()))) {
```

Comment:
Something wrong
`(!X && !Y) || (X && !Y)`
`(!X || X) && !Y`
`!Y`

Reply:
Hm, I tested all 4 cases we're going for and it works (Though it's far from pretty)

Reply:
I mean `!userSettings.detailsBanner()` is enough :confused: